### ISSUE TITLE
nullpointer exception

### ISSUE THREAD

**Issue by darosior** [bisq-network/bisq#3846]
Just got a `java.lang.NullPointerException`, I'm running v1.2.4 with a local `bitcoind`.
It happened when I clicked on a block chain confirmation notification IIRC.

```
Jan-02 12:49:29.394 [JavaFX Application Thread] INFO  b.c.d.m.DaoStateMonitoringService: ParseBlockChainComplete: Accumulated updateHashChain() calls for 35 block took 1514 ms (43 ms in average / block) 
Jan-02 12:49:30.754 [JavaFX Application Thread] INFO  b.c.b.w.BsqWalletService: updateBsqBalance took 41 ms 
Jan-02 12:49:30.780 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetDaoStateHashesRequest message to peer ef5qnzx6znifo3df.onion:8000 succeeded. 
Jan-02 12:49:30.780 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetDaoStateHashesRequest message to peer jhgcy2won7xnslrb.onion:8000 succeeded. 
Jan-02 12:49:30.780 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetProposalStateHashesRequest message to peer jhgcy2won7xnslrb.onion:8000 succeeded. 
Jan-02 12:49:30.780 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetProposalStateHashesRequest message to peer ef5qnzx6znifo3df.onion:8000 succeeded. 
Jan-02 12:49:30.780 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetBlindVoteStateHashesRequest message to peer jhgcy2won7xnslrb.onion:8000 succeeded. 
Jan-02 12:49:30.780 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetBlindVoteStateHashesRequest message to peer ef5qnzx6znifo3df.onion:8000 succeeded. 
Jan-02 12:49:30.792 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer jhgcy2won7xnslrb.onion:8000 a GetDaoStateHashesResponse with 11 stateHashes 
Jan-02 12:49:30.813 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer jhgcy2won7xnslrb.onion:8000 a GetProposalStateHashesResponse with 5 stateHashes 
Jan-02 12:49:30.814 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer jhgcy2won7xnslrb.onion:8000 a GetBlindVoteStateHashesResponse with 5 stateHashes 
Jan-02 12:49:30.814 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer ef5qnzx6znifo3df.onion:8000 a GetDaoStateHashesResponse with 11 stateHashes 
Jan-02 12:49:30.824 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer ef5qnzx6znifo3df.onion:8000 a GetProposalStateHashesResponse with 5 stateHashes 
Jan-02 12:49:30.824 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer ef5qnzx6znifo3df.onion:8000 a GetBlindVoteStateHashesResponse with 5 stateHashes 
Jan-02 12:49:35.808 [JavaFX Application Thread] INFO  b.c.d.m.DaoStateMonitoringService: Conflict with non-seed nodes: We received a block hash from peer 63kio4v4dgx4ykou.onion:9999 which conflicts with our block hash.
my daoStateHash=StateHash{
     height=610935,
     hash=6a34724386706751725ba3dba80b6e31eb724277,
     prevHash=7d76e07bf7a8b086bc8ea3a2afff82a75363e5a8
}
peers daoStateHash=StateHash{
     height=610935,
     hash=f2e6ceb407bc2b668b3c745b22dedc6c71dbbfcb,
     prevHash=c9e14841e965dff642d68c9ba9c9d92dec64c236
} 
Jan-02 12:49:50.041 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: Uncaught Exception from thread JavaFX Application Thread 
Jan-02 12:49:50.041 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: throwableMessage= null 
Jan-02 12:49:50.041 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: throwableClass= class java.lang.NullPointerException 
Jan-02 12:49:50.043 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: Stack trace:
java.lang.NullPointerException
	at com.jfoenix.skins.JFXTextFieldSkin.updateTextPos(JFXTextFieldSkin.java:109)
	at com.jfoenix.skins.JFXTextFieldSkin.layoutChildren(JFXTextFieldSkin.java:93)
	at javafx.controls/javafx.scene.control.Control.layoutChildren(Control.java:601)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1211)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1218)
	at javafx.graphics/javafx.scene.Scene.doLayoutPass(Scene.java:590)
	at javafx.graphics/javafx.scene.Scene$ScenePulseListener.pulse(Scene.java:2507)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.lambda$runPulse$2(Toolkit.java:410)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.runPulse(Toolkit.java:409)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.firePulse(Toolkit.java:436)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:518)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:498)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:491)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:319)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
	at java.base/java.lang.Thread.run(Thread.java:844)
```

**Comment by dikdust** [comment]
Same for me on non standard ubuntu 18.04 after #3825 with 1.2.5

![image](https://user-images.githubusercontent.com/25555288/72689800-c5605c00-3b15-11ea-84fb-6921bc3ad9fd.png)

gen-19 23:42:24.955 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: Uncaught Exception from thread JavaFX Application Thread 
gen-19 23:42:24.956 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: throwableMessage= null 
gen-19 23:42:24.956 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: throwableClass= class java.lang.NullPointerException 
gen-19 23:42:24.972 [JavaFX Application Thread] ERROR b.c.s.CommonSetup: Stack trace:
java.lang.NullPointerException
        at bisq.desktop.main.overlays.Overlay.layout(Overlay.java:728)
        at bisq.desktop.main.overlays.Overlay.display(Overlay.java:542)
        at bisq.desktop.main.overlays.windows.WalletPasswordWindow.show(WalletPasswordWindow.java:145)
        at bisq.desktop.main.MainViewModel.lambda$setupHandlers$14(MainViewModel.java:319)
        at bisq.core.app.BisqSetup.lambda$initWallet$11(BisqSetup.java:600)
        at bisq.core.app.WalletAppSetup.lambda$init$2(WalletAppSetup.java:162)
        at bisq.common.reactfx.FxTimer.lambda$restart$0(FxTimer.java:93)
        at javafx.graphics/com.sun.scenario.animation.shared.TimelineClipCore.visitKeyFrame(TimelineClipCore.java:239)
        at javafx.graphics/com.sun.scenario.animation.shared.TimelineClipCore.playTo(TimelineClipCore.java:180)
        at javafx.graphics/javafx.animation.Timeline.doPlayTo(Timeline.java:177)
        at javafx.graphics/javafx.animation.AnimationAccessorImpl.playTo(AnimationAccessorImpl.java:39)
        at javafx.graphics/com.sun.scenario.animation.shared.SingleLoopClipEnvelope.timePulse(SingleLoopClipEnvelope.java:99)
        at javafx.graphics/javafx.animation.Animation.doTimePulse(Animation.java:1101)
        at javafx.graphics/javafx.animation.Animation$1.lambda$timePulse$0(Animation.java:186)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at javafx.graphics/javafx.animation.Animation$1.timePulse(Animation.java:185)
        at javafx.graphics/com.sun.scenario.animation.AbstractMasterTimer.timePulseImpl(AbstractMasterTimer.java:344)
        at javafx.graphics/com.sun.scenario.animation.AbstractMasterTimer$MainLoop.run(AbstractMasterTimer.java:267)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:514)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:498)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:491)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:319)
        at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
        at java.base/java.lang.Thread.run(Thread.java:844)
 
gen-19 23:42:25.317 [JavaFX Application Thread] ERROR b.d.app.BisqApp: Error at displaying Throwable.

**Comment by cd2357** [comment]
Should be fixed in the most recent release (v1.6.4) which upgraded several UI components and libraries. Please try it out and let us know if it's still an issue for you.
